tagstore/tagstore.py

```diff
@@ -74,14 +64,22 @@ def _add_tag_to_taxonomy(self, taxonomy_uid: int, tag: str, parent_tag: Optional
         """
         raise NotImplementedError()
 
-    def list_tags_in_taxonomy(self, uid: int) -> Iterator[Tag]:
+    def get_tag_in_taxonomy(self, name: str, taxonomy_uid: TaxonomyId) -> Optional[Tag]:
```

Comment:
Docstring for [`_add_tag_to_taxonomy` above](https://github.com/open-craft/blockstore/pull/15/files#diff-a649213e62430d6f8f65bc09c1007e05R62) should read:

> It should return the 'name' value of the newly created tag, ...